### DISCUSSION TITLE
We need mod compatibility with vanilla or other master servers to play OFFLINE against bots. Bring back old BF2 memories!

### DISCUSSION THREAD

**Discussion by BaumTV** [Joe91/fun-bots#321] [Ideas]
Hey, i would like to use this mod with BF-Portable (mobile AiO LAN server tool for most BF games), but there is one little problem:
I can't use the fun bots mod outside of Venice Unleashed servers. But iam not that kind of an always multiplayer dude, who wants to compete himself everyday with others. I wan't to play against bots like in Battlefield 2 (the first BF game i've ever played).
 
EA doesn't care about it's fanbase but i hope you do. They refused to implemented that in any of their new games, so when the servers shut off, we are all screwed and all what is left are some memories and some useless client/server files. I know i can play on VU, but already as i mentioned, i was comming from BF2 and it's a shame that i can't play any of the newer games on LANs offline with bots.

I don't know anything about coding, so iam pretty screwed when official servers and VU is shutting down.

Iam speaking deeply out of my heart and i hope my dream of this will someday becoming true. 

Kind regards
_An Battlefield Fan_

You will find the files for the BF-Portable tool here: https://shorturl.at/cerS1

**Comment by koopa990** [comment]
I think you're reaching a little too far

**Reply by BaumTV** [reply]
No iam not. This is pure ignorance. DICE excluded bot support for newer games for maximizing their damn profit. When they shut down you all are screwed and need to play the newest titles wich you maybe not like to play. This is the same behaviour EA has to it's community and this is really just pure arrogance and ignorance. Where are the good times? We want bot support back and it needs to be also offline usable!

**Comment by Firjens** [comment]
Hi,

Feel free to fork this repository and adapt it to your linking according to the license.

**Reply by BaumTV** [reply]
Done, but i can't adapt anything, because i don't know anything about coding...

**Reply by koopa990** [reply]
> Done, but i can't adapt anything, because i don't know anything about coding...

What's the point of making this thread, stating you about coding.

Nobody is going to make this work off VU for a long time

**Reply by BaumTV** [reply]
Some good old BF player will. Thanks for the support, i believe someone will do it. Art (game) preservation is a good thing to keep everything alive and i think all people know that VU will not last forever. I love archiving and iam archiving A LOT. All of the Battlefield titles are no exeption. I hope someday i can add the working bot files to my archive and play offline with my friends in LAN against bots.

**Comment by koopa990** [comment]
> Some good old BF player will. Thanks for the support, i believe someone will do it. Art (game) preservation is a good thing to keep everything alive and i think all people know that VU will not last forever. I love archiving and iam archiving A LOT. All of the Battlefield titles are no exeption. I hope someday i can add the working bot files to my archive and play offline with my friends in LAN against bots.

Yeah, well, maybe you should focus on WR too.

**Reply by BaumTV** [reply]
> > Some good old BF player will. Thanks for the support, i believe someone will do it. Art (game) preservation is a good thing to keep everything alive and i think all people know that VU will not last forever. I love archiving and iam archiving A LOT. All of the Battlefield titles are no exeption. I hope someday i can add the working bot files to my archive and play offline with my friends in LAN against bots.
> 
> Yeah, well, maybe you should focus on WR too.

I should do my own mod. Iam just going back to BFBC2.
